EDI/Integration/01_ManageConnections.md

```diff
@@ -0,0 +1,153 @@
+[!!CONNECTIONS (Settings)](../UserInterface/02a_Connections.md)
+
+# Manage connections
+
+Establish connections via drivers to connect your EDIFACT message receivers with the *Actindo Core1 Platform*. The drivers represent a specific internet protocol to transfer the data.   
+In the following, the general settings to create, edit, enable, and disable connections for EDIFACT messages are described.
+
+## Create connection
+
+Create a connection to a certain EDIFACT message receiver. 
+
+#### Prerequisites
+
+- At least one driver has been installed.
+- Check with your message receiver the protocol to be used.
+
+#### Procedure
+
+*DataHub Exporter > Settings > Tab CONNECTIONS*
+
+![Connection](../../Assets/Screenshots/EDI/Integration/Connections.png "[Connection]")
+
+1. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.   
+    The *Create connection* view is displayed.
+
+    ![Create connection](../../Assets/Screenshots/EDI/Integration/CreateConnections.png "[Create connection]")
+
+2. Enter a name for the connection in the *Name* field. It might be useful to choose a name that contains a precise description of the purpose, so that you have more flexibility for further connection names, for example, "EDIFACT to [supplier]". 
+
+3. Click the *Driver* drop-down list and select the driver/protocol your business partner require. For EDIFACT messages, in most cases, the SFTP or the AS2 drivers are required.   
+    The following drivers are available in the standard:    
+        - SFTP   
+        - Internal  
+        - POST call   
+        - Output
+    > [Info] For the AS2 (Applicability Statement 2) protocol, the *AS2Transport* plugin is available in the *Core1 Platform*. Contact your MPS consultant if your business partner requires this protocol.
+     
+    The *Credentials* section is displayed below the *Driver* field.
+
+    ![Credentials](../../Assets/Screenshots/EDI/Integration/Credentials.png "[Credentials]")
+
+    > [Info] Depending on the selected driver, the fields in the *Credentials* section differ.
+
+4. Enter the required data in the *Credentials* section fields. Use the information you received from the business partner to fill the credentials.
+
+5. Click the [SAVE] button.   
+    The connection will be established. The *Checking credentials...* notice is displayed. After a few seconds, the *Sync triggered successfully* window is displayed.  
+    The *Connections* view is displayed, the newly created connection is added. After successful synchronization, the entry is activated automatically. You may have to refresh the application to see the *Active* status.
+
+ 6. If necessary, click the ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh) button to update the list of connections.   
+    The status of the connection in the *Status* column has switched to **Active**.
+
+
+
```

Comment:
Create connections, vor allem ab Edit connection, sind sehr ähnlich wie die Connections in Omni-Channel. Sinnvoll, hier darauf zu verweisen? Oder eher in einer zweiten/zukünftigen Version, wenn wir die Verbindung-Erstellung "verallgemeinen" und alle "Establish connection" darauf verweisen...

Reply:
Lieber warten, bis wir es in Core 1 haben